packages/components/src/profile/profile_tag/index.tsx

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {useTeammateNameDisplaySetting, useUser} from '../../hooks';
+
+import ProfileTag from './profile_tag';
+
+interface Props {
+    userId: string;
+}
+
+const ProfileTagConnector = (props: Props) => {
+    const user = useUser(props.userId);
+    const teamnameNameDisplaySetting = useTeammateNameDisplaySetting();
+
+    return (
+        <ProfileTag
```

Comment:
For components that need to be connected to redux, I put all redux connections in a higher order component and left the actual components ignorant of redux or where the props come from.

Reply:
Thanks for doing that, but I do think we'll want to leave that logic out of this package for now